docs/topics/auth/customizing.txt

```diff
@@ -772,6 +772,11 @@ utility methods:
 
     .. method:: models.BaseUserManager.make_random_password(length=10, allowed_chars='abcdefghjkmnpqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ23456789')
 
+        .. deprecated:: 4.2
+            The ``make_random_password`` method is deprecated and will be
+            removed in Django 5.0 as it's not used by Django and the generated
+            passwords don't comply with ``AUTH_PASSWORD_VALIDATORS``.
```

Comment:
```suggestion
            The ``make_random_password`` method is deprecated and will be
            removed in Django 5.1.
```
Not sure we need to go into the details of why we are deprecating it :) That might be misleading for the purpose of the auth docs.